README.md

```diff
@@ -132,57 +132,56 @@ curl -u "YOUR_LAMBDATEST_USERNAME:YOUR_LAMBDATEST_ACCESS_KEY" -X POST "https://m
 
 ## Run Your First Test
 
-**Test Scenario:** Check out [Android.rb](https://github.com/LambdaTest/LT-appium-ruby-cucumber/blob/master/Android/examples/First_test/features/step_definitions/first_steps.rb) file to view the sample test script for android and [iOS.rb](https://github.com/LambdaTest/LT-appium-ruby-cucumber/blob/master/iOS/examples/First_test/features/step_definitions/first_steps.rb) for iOS.
+**Test Scenario:** Check out [first_steps.rb](https://github.com/LambdaTest/LT-appium-ruby-cucumber/blob/master/Android/examples/First_test/features/step_definitions/first_steps.rb) file to view the sample test script in android/ios folder and [first.config.yml](https://github.com/LambdaTest/LT-appium-ruby-cucumber/blob/master/iOS/examples/First_test/features/step_definitions/first_steps.rb) to make changes to the username, accessKey and app url.
 
 ### Configuring Your Test Capabilities
 
 You can update your custom capabilities in test scripts. In this sample project, we are passing platform name, platform version, device name and app url (generated earlier) along with other capabilities like build name and test name via capabilities object. The capabilities object in the sample code are defined as:
 
 <Tabs className="docs__val">
 
-<TabItem value="ios-config" label="iOS" default>
+<TabItem value="android-config" label="Android" default>
+
+```ruby title="Android first.config.yml)"
+server: "mobile-hub.lambdatest.com"
+user: "LT_USERNAME"    # Add Lambdatest username here
+key: "LT_ACCESS_KEY"   # Add Lambdtest accessKey here
 
-```ruby title="iOS(.ipa)"
-server: "beta-hub.lambdatest.com"
 
 common_caps:
-  "user": "LAMBDATEST_USERNAME"
-  "accessKey": "LAMBDATEST_ACCESS_KEY"
-  "project": "First Cucumber Android Project"
-  "build": "build-1"
 
+  "build": "br-1"
+  
 
 browser_caps:
   -
-    "platformName": "iOS"
-    "platformVersion": "12.0"
-    "deviceName": "iPhone 8"
-    "app": "YOUR_APP_URL"
-    "name": "first_test"
     "isRealMobile": "true"
+    "platform": "android"
+    "deviceName": "Galaxy S21 5G"
+    "platformVersion": "11"
+    "app": "APP_URL"      # Add your app url here
 ```
 
 </TabItem>
-<TabItem value="android-config" label="Android" default>
+<TabItem value="ios-config" label="iOS" default>
 
-```ruby title="Android(.apk)"
-server: "beta-hub.lambdatest.com"
+```ruby title="iOS first.config.yml"
+server: "mobile-hub.lambdatest.com"
+user: "LT_USERNAME"    # Add Lambdatest username here
+key: "LT_ACCESS_KEY"   # Add Lambdtest accessKey here
 
 common_caps:
-  "user": "LAMBDATEST_USERNAME"
-  "accessKey": "LAMBDATEST_ACCESS_KEY"
-  "project": "First Cucumber Android Project"
-  "build": "build-1"
 
+  "build": "br-2"
+  
 
 browser_caps:
   -
-    "platformName": "Android"
-    "platformVersion": "9"
-    "deviceName": "Galaxy S8 Plus"
-    "app": "YOUR_APP_URL"
-    "name": "first_test"
     "isRealMobile": "true"
+    "platform": "ios"
+    "deviceName": "iPad (2017)"
```

Comment:
Lets not use iPad, instead use iPhone 11 iOS 14

Reply:
Done